install/update_91_92.php

```diff
@@ -625,6 +598,7 @@ function update91to92() {
                                                 'login_remember_default' => 1));
 
    if (TableExists('glpi_bookmarks')) {
+      // yllen: you backup an inexisting table. If backup existing table it's done at the beginning
```

Comment:
Totally agree here, this seems wrong.
Once the table is rename from "glpi_bookmarks", "glpi_savedsearches", you MUST keep it.
How can both exists ?